.github/workflows/labeler.yml

```
@@ -0,0 +1,59 @@
+name: Smoke check
+run-name: 'Label - ${{ format("PR #{0}", github.event.pull_request.number) }}'
```

actionlint

got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)